### DISCUSSION TITLE
Not receiving 2FA SMS under setup

### DISCUSSION THREAD

**Discussion by yanislavzagorov** [community/community#37578] [Other Feature Feedback, Questions, & Ideas]
I am trying to activate 2FA on my account using [the following page](https://github.com/settings/two_factor_authentication/setup/intro), but after clicking on "Send authentication code", a loader spins up and never stops spinning.

Have tried waiting for 15+ minutes - nada. The loader just sits there, _menacingly_. 

Located in Norway and using a Norwegian phone number, never had issues previously.

Known issue?

**Comment by ossanna16** [comment]
Hi there @yanislavzagorov and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
